scrapers/base.py

```diff
@@ -43,3 +44,24 @@ def get_html_content(self, url):
             print "Url is not reachable. Error logged on: {}".format(
                 strftime("%Y-%m-%d %H:%M:%S", gmtime()))
             log.info(str(err))
+
+    def local_store(self, data):
+        """Writes content from the website to file.
+        Usage::
+             the file name to write the data
+        :param_train_data: the filename
+        :rtype: outputs data to file.
+        """
+        file_name = "news"
+        if file_name:
```

Comment:
I don't understand why we check if `file_name` exists first despite setting it just above. Is there a reason for this?

Reply:
Noted.